resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -32,15 +44,50 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String fname = ServletHelpers.getParameter(request, "fname", "");
     String lname = ServletHelpers.getParameter(request, "lname", "");
     String email = ServletHelpers.getParameter(request, "email", "");
-    reviewee = new Reviewee(fname, lname, email);
+    String resumeURL = getUploadedFileUrl(request, "resume");
+    reviewee = new Reviewee(fname, lname, email, resumeURL);
+
     Entity revieweeEntity = new Entity("Reviewee");
     revieweeEntity.setProperty("first-name", fname);
     revieweeEntity.setProperty("last-name", lname);
     revieweeEntity.setProperty("email", email);
+    revieweeEntity.setProperty("resumeURL", resumeURL);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
-    response.sendRedirect("resume-review.html");
+    // response.sendRedirect("resume-review.html");
+    response.sendRedirect("reviewee-form.html");
```

Comment:
Redirecting to reviewee-form.html will just append that to current URL path and that's why you get `/_ah/upload/reviewee-form.html` path which causes Error 405. Instead redirect it to `/resume-review.html` to move to next page. Using `/` will write URL path from root.

Reply:
Thanks Ani, this worked!